djangocms_pageadmin/test_utils/factories.py

```diff
@@ -112,3 +112,13 @@ def version(self, create, extracted, **kwargs):
             # Simple build, do nothing.
             return
         PageVersionFactory(content=self, **kwargs)
+
+
+class PageUrlFactory(factory.django.DjangoModelFactory):
+    slug = ''
```

Comment:
```suggestion
    slug = factory.Faker("slug")
```
This will auto generate a random slug for you but still allow you to pass in a custom one. I'm less sure about what to use for `path` but since it is a nullable field on the model, you could not define it here.